### ISSUE TITLE
Please do not disable console.log

### ISSUE THREAD

**Issue by crucialfelix** [pchomik/linter-python#23]
It seems that this package is disabling console.log for everything in Atom. The message posted to the console (just before it does it) suggests that just Linter-Python will not print debug; but it doesn't suggest to me that it's going to just hack off console.log and affect all other packages in Atom.

```
            console.log("[Linter-Python] Debug prints were disabled.");
            console.log = () => {};
```

It would be much better if you could change your implementation so that enableDebug just affects debugs posted from linter-python.

Thanks,

**Comment by pchomik** [comment]
Hi,

You are right. This is a hack and I missed the possible impact. How critical is your request? 
I'm asking because I'm in the middle of refactoring and I would like to finish it without middle work. Could you wait week for such fix?

**Comment by crucialfelix** [comment]
I just turned on enableDebug, so it is fine for me now. It was just frustrating because it took a long time to figure out why console.log did nothing.

**Comment by pchomik** [comment]
Change available in 3.0.4
